modules/fts/pages/fts-consistency-vectors.adoc

```diff
@@ -0,0 +1,63 @@
+= Consistency Vectors
+
+Consistency Vectors: The value of the scan_vector parameter.
+
+Scan vectors are built of two-element [value, guard] entries:
+
+value: a vBucket’s sequence number (a JSON number)
+
+guard: a vBucket’s UUID (a string)
+
+Scan vectors have two forms:
+
+Full scan vector: an array of [value, guard] entries, giving an entry for every vBucket in the system.
+
+Sparse scan vectors: an object providing entries for specific vBuckets, mapping a vBucket number (a string) to each [value, guard] entry.
+
+Example of a Sparse Scan Vector
+{ "5 ": [5409393,"VB5ID"], "19": [47574574, "VB19ID"] }
```

Comment:
Like i mentioned already - this is not the syntax we support on server.
The example below captures the right syntax.